tests/php/src/Admin/UserRESTEndpointExtensionTest.php

```diff
@@ -68,7 +68,18 @@ public function test_update_review_panel_dismissed_for_template_mode() {
 		$server = rest_get_server();
 		$this->user_rest_endpoint_extension->register_rest_field();
 
-		$admin_user  = self::factory()->user->create_and_get( [ 'role' => 'administrator' ] );
+		$admin_user = self::factory()->user->create_and_get( [ 'role' => 'administrator' ] );
+
+		if ( is_multisite() ) {
+			$admin_user = self::factory()->user->create_and_get(
+				[
+					'role' => 'administrator',
+				]
+			);
+
+			grant_super_admin( $admin_user->ID );
+		}
+
```

Comment:
This should also use `MockAdminUser`

Reply:
@westonruter Please note the use of `create_and_get` and later this test also requires the WP user object.

Reply:
You could make `MockAdminUser` return the object to use later.